app/assets/javascripts/services/alerts_center_service.js

```diff
@@ -606,7 +606,7 @@ angular.module('ManageIQ').service('alertsCenterService', ['API', '$q', '$timeou
             providerType = getObjectType(provider);
             descriptors = provider.type.toLowerCase().split('::');
             if (descriptors.length >= 3) {
-              summaryItem.displayType = descriptors[1];
+              summaryItem.displayType = __(descriptors[1]);
```

Comment:
I removed everything related to translating the tags and also found the other place "providers" was coming from. Unfortunately, provider.type is coming from here (https://github.com/ManageIQ/manageiq-ui-classic/blob/e91c151eff240beeeabdc75d950002612c33ff56/app/assets/javascripts/services/alerts_center_service.js#L367-L370) and I don't think I should mark things in the API for translation. Thoughts? @himdel @mzazrivec

Reply:
For reference, `provider.type` would look something like this `type: "ManageIQ::Providers::Redhat::NetworkManager"`

Reply:
Aaah... if that's the case, we should be able to use the `display_name` method.

In ruby code...

```
> ManageIQ::Providers::Redhat::NetworkManager.display_name
=> "Network Manager"
```

(that's a string in the catalog, so `_()` can be applied safely, in ruby or in js)

Cc @abellotti , @skateman  .. sounds like we need `display_name` in the providers api.

Reply:
Don't we rather need a `ui_lookup` implementation in JS? cc @mzazrivec

Reply:
@skateman I don't know how would such a thing work. IMO the nice-looking string should come from API.